controller/ucp/subscriptions.php

```diff
@@ -184,7 +184,7 @@ protected function display_sections()
 					if (!$row['contrib_id'])
 					{
 						$this->delete_subscription($row['watch_object_type'], $row['watch_object_id'], false);
-						continue;
```

Comment:
Please make sure this works on 7.3

Reply:
Yes. In fact 7.3 would issue a warning here saying we should be using `continue 2`

Reply:
Ok, just wanted to be sure :)